sources/packages/backend/apps/api/src/route-controllers/announcements/announcements.controller.ts

```diff
@@ -0,0 +1,20 @@
+import { Controller, Get } from "@nestjs/common";
+import BaseController from "../BaseController";
+import { ApiTags } from "@nestjs/swagger";
+import { Announcements } from "@sims/sims-db";
+import { AnnouncementsService } from "../../services";
+import { Public } from "../../auth/decorators/public.decorator";
+
+@Controller("announcements")
```

Comment:
we have always followed the singular can convention to name the controller root. In this case, `announcement`

Reply:
Also, same applies for the file name. `assnouncement.controller.ts`

Reply:
file names updated